Sources/Views/AnimatableTableViewCell.swift

```diff
@@ -156,7 +159,9 @@ open var startPoint: GradientStartPoint = .top
   fileprivate func configureInspectableProperties() {
     configureAnimatableProperties()
     configureOpacity()
+    #if os(iOS)
```

Comment:
I'll see what I can do to hide some things and then we can compare the two approaches.